clients/cli/src/main/java/org/apache/gravitino/cli/GravitinoOptions.java

```diff
@@ -65,7 +66,7 @@ public Options options() {
     options.addOption(createArgOption("m", METALAKE, "metalake name"));
     options.addOption(createSimpleOption("i", IGNORE, "ignore client/sever version check"));
     options.addOption(createSimpleOption("a", AUDIT, "display audit information"));
```

Comment:
"display audit information", "Display index information"; Some of the comment are "display", some are "Display", can you make them consistent?

Reply:
can do - although I think that was fixed in another PR/may be a merge issue

Reply:
can you fix it now? I don't think it is a huge work.

Reply:
Sure and done.